### ISSUE TITLE
fundchannel consumes all of the utxo, regardless of how much of the utxo is still available

### ISSUE THREAD

**Issue by jl777** [ElementsProject/lightning#221]
The title pretty much describes it.
Since there is little to no visiblity into the change txid or currently active addfunds, it seems for now we just need to do fundchannel with 100% of the funds in each utxo?

If so, that creates some issues...
hopefully there will be a way to get a list of the currently active utxos, how much each one has available outside of funded channels, etc.

**Comment by cdecker** [comment]
Yeah, this is a known bug. I forgot to re-add the change output to the database, I'll address this ASAP
